core/systems/performance/performance_manager.gd

```diff
@@ -562,7 +565,7 @@ func _set_sane_defaults() -> void:
 ## Reads the pp_od_clk_voltage from sysfs and returns the OD_RANGE values. This file will 
 ## be empty if not in "manual" for pp_od_performance_level.
 func _read_amd_gpu_clock_limits() -> void:
-	var args := ["/sys/class/drm/card0/device/pp_od_clk_voltage"]
+	var args := ["/sys/class/drm/" + gpu.card.name + "/device/pp_od_clk_voltage"]
```

Comment:
Is it possible for this to ever be null? Do we have a fallback path?

Reply:
We could set it to card1 by default. If that is incorrect though it won't work anyway and we have a bigger issue that detection didn't work.